spec/services/google/keyword_service_spec.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Google::KeywordService, type: :service do
+  describe '#call' do
+    context 'given a keyword with valid result' do
+      it 'returns 3 top ads count' do
+        stub_request(:get, 'https://www.google.com/search?gl=en&hl=en&q=vpn').to_return(body: file_fixture('html/valid_google.html').read)
+        result = described_class.new('vpn').call
+        expect(result[:ads_top_count]).to eq(3)
+      end
+
+      it 'returns 4 total ads count' do
+        stub_request(:get, 'https://www.google.com/search?gl=en&hl=en&q=vpn').to_return(body: file_fixture('html/valid_google.html').read)
+        result = described_class.new('vpn').call
+        expect(result[:ads_page_count]).to eq(4)
+      end
+
+      it 'returns the correct ads urls' do
+        stub_request(:get, 'https://www.google.com/search?gl=en&hl=en&q=vpn').to_return(body: file_fixture('html/valid_google.html').read)
+        result = described_class.new('vpn').call
+        expect(result[:ads_top_urls]).to contain_exactly('https://hubstaff.com/features/automatic-time-tracker', 'https://hubstaff.com/time_tracking_software', 'https://hubstaff.com/pricing', 'https://hubstaff.com/demo', 'https://www.odoo.com/app/timesheet', 'https://zapier.com/apps/timely-time-tracking/integrations')
+      end
+
+      it 'returns 10 results count' do
```

Comment:
```suggestion
      it 'counts 10 results' do
```
An alternative that works too 😇

Reply:
What do you think about `returns results count of 10` so that we can keep the `returns {value}` format 🙏

Reply:
Perfectly fine as well! :)